knative-operator/pkg/controller/knativekafka/knativekafka_controller.go

```diff
@@ -57,7 +60,59 @@ func add(mgr manager.Manager, r reconcile.Reconciler) error {
 		return err
 	}
 
-	// NOT IMPLEMENTED YET
+	err = c.Watch(&source.Kind{Type: &corev1.ConfigMap{}}, &handler.EnqueueRequestForOwner{
+		OwnerType:    &operatorv1alpha1.KnativeKafka{},
+		IsController: true,
+	})
+	if err != nil {
+		return err
+	}
```

Comment:
Why is this necessary? Are we creating any configmaps specifically that wouldn't be caught by the generic loop below?

Reply:
The config map for the kafka bootstrap server (backed by the CRD)?

Reply:
Nah, that will be caught by the generic loop below.

This is a copy paste error